#!/usr/bin/env python
PACKAGE = "ares_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("controller_param_p", double_t, 0, "Path following controller PID gain, p",    1.00,   -100.0,   100.0)
gen.add("controller_param_i", double_t, 0, "Path following controller PID gain, i",    50.0,   -100.0,   100.0)
gen.add("controller_param_d", double_t, 0, "Path following controller PID gain, d",    0.50,   -100.0,   100.0)

exit(gen.generate(PACKAGE, "ares_control", "PathController"))
